cmd/system-probe/config/config.go

```diff
@@ -142,13 +144,15 @@ func load(configPath string) (*Config, error) {
 		StatsdHost: aconfig.GetBindHost(),
 		StatsdPort: cfg.GetInt("dogstatsd_port"),
 
-		ProfilingEnabled:     cfg.GetBool(key(spNS, "internal_profiling.enabled")),
-		ProfilingSite:        cfg.GetString(key(spNS, "internal_profiling.site")),
-		ProfilingURL:         cfg.GetString(key(spNS, "internal_profiling.profile_dd_url")),
-		ProfilingAPIKey:      aconfig.SanitizeAPIKey(cfg.GetString(key(spNS, "internal_profiling.api_key"))),
-		ProfilingEnvironment: cfg.GetString(key(spNS, "internal_profiling.env")),
-		ProfilingPeriod:      cfg.GetDuration(key(spNS, "internal_profiling.period")),
-		ProfilingCPUDuration: cfg.GetDuration(key(spNS, "internal_profiling.cpu_duration")),
+		ProfilingEnabled:        cfg.GetBool(key(spNS, "internal_profiling.enabled")),
```

Comment:
Can we please put all of these "reads from config" into the `profiling.Start` function. They all use the same config and it's getting extremely repetitive. It's shared, global and universal for all agents. No need to repeatedly write this code in each one. Or, am I missing something?

Reply:
> It's shared, global and universal
Alas, no, no and no. Different agents handle their configration differently; profiling settings are stored under different configuration prefixes, they don't even have the same settings (core notably has less). I agree that this could be improved, but that deserves at least a separate PR.

Reply:
Fine by me.